core/src/integration/java/com/radixdlt/integration/steady_state/deterministic/consensus/OneProposalTimeoutResponsiveTest.java

```diff
@@ -104,24 +104,28 @@ private void run(int numValidatorNodes, long numRounds, long dropPeriod) {
     test.runUntilMessage(DeterministicTest.hasReachedRound(Round.of(numRounds)));
 
     long requiredIndirectParents =
-        numValidatorNodes <= 3
-            ? 0 // there are no indirect parents for 3 nodes (QC is always formed)
+        numValidatorNodes <= 4
```

Comment:
the change is because the leader receives their own proposal and has their round extended, while others form a timeout QC (instead of a TC from mixed proposal and empty vertex votes)